doc/ref_model/chapters/chapter04.md

```diff
@@ -280,45 +280,77 @@ The idea of the Cloud Infrastructure profiles is to have a predefined set of inf
-| .tiny                  | 1                        | 512 MB                  | 1 GB             | 1 Gbps               |
-| .small                 | 1                        | 2 GB                    | 20 GB            | 1 Gbps               |
-| .medium                | 2                        | 4 GB                    | 40 GB            | 1 Gbps               |
-| .large                 | 4                        | 8 GB                    | 80 GB            | 1 Gbps               |
-| .2xlarge <sup>1)</sup> | 8                        | 16 GB                   | 160 GB           | 1 Gbps               |
-| .4xlarge <sup>1)</sup> | 16                       | 32 GB                   | 320 GB           | 1 Gbps               |
-| .8xlarge <sup>1)</sup> | 32                       | 64 GB                   | 640 GB           | 1 Gbps               |
-
-<p align="center"><b>Table 4-13:</b> Predefined Compute Flavours.</p>
-
-**1)** These compute Flavours are intended to be used for transitional purposes and workload vendors are expected to consume smaller Flavours and adopt microservices-based designs for their workloads.<br>
-**2)** In Kubernetes based environments these are the resource requests of the containers in the pods. To get guaranteed resources the resource requests should be set to the same values as the resource limits, to get burstable resources the resource limits should be higher than the resource requests while to get best effort resources none of resource requests of resource limits should be set.<br>
-**3)** The "local disk" is an ephemeral disk that provides storage for the life of a VM/Pod, and can either be provided by storage devices housed within the physical server on which the VM/Pod is running, or from an external storage device such as SAN or NFS.
+
+| Ref | Attribute | Unit | (B)asic/(H)igh Performance | Presence |
+|----------|--------------------------------------------|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| -----------|
+| i.fl.001 | Flavor ID             | integer       | B/H           | Required |
+| i.fl.002 | vCPU                      | integer      | B/H           | Required |
+| i.fl.003 | RAM                       | Gb           | B/H           | Required |
+| i.fl.004 | external_disk             | Gb           | B/H           | Required |
+| i.fl.005 | ephemeral-Disk            | Gb           | B/H           | Optional |
+| i.fl.006 | root-disk                 | GB           | H             | Required
+| i.fl.007 | cpu_SMT                   | Y/N          | H             | Optional
+| i.fl.008 | cpu_pinning               | Y/N          | H             | Optional
+| i.fl.009 | cpu_alloc_ratio           | n:m          | H             | Optional
```

Comment:
CPU Allocation Ratio for H is 1:1 for B will need to be specified or there should be a default

Reply:
```suggestion
| i.fl.009 | cpu_alloc_ratio           | 1:1/n:m          | B/H             | Optional
```